_episodes_rmd/10-functions.Rmd

```diff
@@ -7,12 +7,14 @@ questions:
 objectives:
 - "Define a function that takes arguments."
 - "Return a value from a function."
+- "Check argument conditions with `stopifnot` in functions."
 - "Test a function."
 - "Set default values for function arguments."
 - "Explain why we should divide programs into small, single-purpose functions."
 keypoints:
 - "Use `function` to define a new function in R."
 - "Use parameters to pass values into functions."
+- "Use `stopifnot` to flexibly check function arguments in R."
 - "Load functions into programs using `source`."
```

Comment:
While you're at it, can you add `()` here too... after `source` as well as `stopifnot`